README.md

```diff
@@ -1,58 +1,70 @@
 # Google - Partnership
 Repository to store the work done by Google Fellows during 2019 
 
-## Workflow for locally staging this work
+* This project uses [yarn](http://yarnpkg.com/) for dependency management. 
+Install and run `yarn install` to install all dependencies. If you ever see
+an issue with missing dependencies, try running this command again.
+
+## Making changes and locally staging
+* Go to the [Firestore database](
+https://console.firebase.google.com/project/mapping-crisis/database/firestore/data~2FALLOWED_USERS~2FALL_USERS)
+while logged in as `gd-earthengine-user@givedirectly.org` and add your Google
+account's email address to the list of users. This will give you access to
+user-drawn features on the disaster map.
+
 * Download this git repository. You will probably have to add ssh keys to your
 account so that you can access this repository. Googling "github add ssh keys"
 is good enough. [Current instructions](https://help.github.com/en/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
 available. However, if you work for Google and get a failed command when you run
 `ssh-add`, try specifying `/usr/bin/ssh-add`, since there is a Google-specific
 `ssh-add` binary that otherwise gets invoked.
 
-* Go to the [Firestore database](
-https://console.firebase.google.com/project/mapping-crisis/database/firestore/data~2FALLOWED_USERS~2FALL_USERS)
-when logged in as `gd-earthengine-user@givedirectly.org` and add your Google
-account's email address to the list of users.
+* Start the local web server by running `yarn run ws --directory docs --port 8080`.
+and visit the page at `localhost:8080` (we recommend in Chrome). 
+
+* In order to view all features without incurring server traffic expenses, 
+[sign up](https://earthengine.google.com/) to whitelist yourself with earth engine
+(usually takes ~1-2 business days to get approved).
 
-* Install [yarn](http://yarnpkg.com/) and run `yarn install` to install all
-dependencies.
+* Install `clang-format`, probably using [`brew install clang-format`](https://brew.sh).
 
-* Start the local web server by running
-`yarn run ws --directory docs --port 8080`.
+* Set up auto-lint: `echo ./pre-push-hook.sh > .git/hooks/pre-push`
 
-* Set `export GOOGLE_APPLICATION_CREDENTIALS=/path/to/secret.json` in your
-shell, most likely in your `~/.bashrc`. Get the service account .json file,
-either from a collaborator or by logging into the [Service Accounts page](
+* *[Optional]* Run `./lint.sh --fix` to run clang-format and eslint on all relevant files and format in
+place. Clang-format will run automatically on commit and eslint will run automatically on push so
```

Comment:
I don't think this is quite right. clang-format doesn't run on commit, but running it then requires you to recommit. Maybe rephrase?

Reply:
Updated. Let me know what you think